central/image/datastore/datastore_impl.go

```diff
@@ -125,16 +126,22 @@ func (ds *datastoreImpl) ListImage(ctx context.Context, sha string) (*storage.Li
 
 // CountImages delegates to the underlying store.
 func (ds *datastoreImpl) CountImages(ctx context.Context) (int, error) {
-	if ok, err := imagesSAC.ReadAllowed(ctx); err != nil {
-		return 0, err
-	} else if ok {
-		return ds.storage.Count(ctx)
+	if !features.PostgresDatastore.Enabled() {
```

Comment:
We shouldn't do this, instead we should fix the images store to do what all the other stores do for sac scopes